app/services/green_lanes/categorisations_service.rb

```diff
@@ -0,0 +1,9 @@
+module GreenLanes
+  class CategorisationsService
+    def find_possible_categorisations(subheading)
```

Comment:
if we rename the service to `FindCategorisationService` this method must be renamed to `call`

Reply:
Done